Assassinations/Schinese/Setup-Assassinations.tra

```diff
@@ -30,11 +30,13 @@
 @29 = ~Э������~
 @30 = ~�����������ݴ�������Ůʿ������������˾�ʿ��ıɱ�ĵ����͸�<CHARNAME>�ġ�Ȼ������Ůʿ���˾ܾ������������ִ������κ���ϵ��
 
-����������
-�����Ȼ���
-����������
-���߿�����
-�������ඨ����~
+�������ϣ�
+
+װ��������
```

Comment:
If you applied EE-specific format of items, you should you Setup-Assassinations-EE,tra file for this. See Polish language file for reference.